lib/history.sh

```diff
@@ -51,10 +46,14 @@ bind '"\e[B": history-search-forward'
 bind '"\e[C": forward-char'
 bind '"\e[D": backward-char'
 
+# Use standard ISO 8601 timestamp
+# %F equivalent to %Y-%m-%d                                                                                      
+# %T equivalent to %H:%M:%S (24-hours format)
+
 # Show history
-#case $HIST_STAMPS in
-  #"mm/dd/yyyy") alias history='fc -fl 1' ;;
-  #"dd.mm.yyyy") alias history='fc -El 1' ;;
-  #"yyyy-mm-dd") alias history='fc -il 1' ;;
-  #*) alias history='fc -l 1' ;;
-#esac
+case $HIST_STAMPS in
+  "mm/dd/yyyy") HISTTIMEFORMAT=$'\033[31m[%m/%d/%Y] \033[36m[%T]\033[0m ' ;;
+  "dd.mm.yyyy") HISTTIMEFORMAT=$'\033[31m [%d.%m.%Y] \033[36m[%T]\033[0m ' ;;
```

Comment:
Is there a reason for a space here?

```suggestion
  "dd.mm.yyyy") HISTTIMEFORMAT=$'\033[31m[%d.%m.%Y] \033[36m[%T]\033[0m ' ;;
```